config/crd/bases/configuration.konghq.com_kongvaults.yaml

```diff
@@ -172,6 +172,9 @@ spec:
         required:
         - spec
         type: object
+        x-kubernetes-validations:
+        - message: The spec.prefix field is immutable
+          rule: self.spec.prefix == oldSelf.spec.prefix
```

Comment:
👍 Please consider adding a test case to the existing `TestCRDValidations` that will verify this works as expected.

Reply:
Added a test case in `TestCRDValidations` in envtest.